### PR TITLE
Test index delete

### PR DESCRIPTION
**Pull request by mvr320**
Edit and Add of new entity will be handled in another PR.

### CODE REVIEW

File: webapp/tests/Controller/Jury/ContestControllerTest.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+namespace App\Tests\Controller\Jury;
+
+use App\Entity\Contest;
+
+class ContestControllerTest extends JuryControllerTest
+{
+    protected static $baseUrl         = '/jury/contests';
+    protected static $shortTag        = 'contest';
+    protected static $deleteEntities  = array('name' => ['Demo practice session']);
```
Comment (nickygerritsen) [inline webapp/tests/Controller/Jury/ContestControllerTest.php:11]: Please use short array syntax (`[]`)

**Review by nickygerritsen — COMMENTED** [review]
Can you format using PSR2? Can be done in phpstorm

File: webapp/tests/Controller/Jury/ContestControllerTest.php (hunk 1 above)
Comment (mvr320) [reply webapp/tests/Controller/Jury/ContestControllerTest.php:11]: I'm under the impression that my editor is in PSR12 mode.. will re-check.

**Comment by mvr320** [comment]
> Can you format using PSR2? Can be done in phpstorm

I'm under the impression that my editor is in PSR12 mode.. will re-check.

File: webapp/tests/Controller/Jury/ContestControllerTest.php (hunk 1 above)
Comment (mvr320) [reply webapp/tests/Controller/Jury/ContestControllerTest.php]: done

**Review by nickygerritsen — APPROVED** [review]

**Pull request merged**